packages/metal-incremental-dom/src/incremental-dom.js

```diff
@@ -666,9 +666,7 @@ import { globals } from 'metal';
    * @param {?Object<string, !Element>} keyMap
    */
   var removeChild = function (node, child, keyMap) {
-    if (child.parentNode === node) {
```

Comment:
This was actually an important fix that they did in incremental-dom. Without it some places throw errors. It's just making sure that `removeChild` won't be called if the given node is not the parent. I don't think we should remove that.